### ISSUE TITLE
How to get application logs from a Function App in linux

### ISSUE THREAD

**Issue by sjuarezgx** [Azure/azure-functions-host#7935]
Hello

I'm working with Net 6, Azure Functions V4 in linux.
I do not use containers.

I need to get the logs generated from the application, but cannot find them. I read the documentation about it, but I don't find it clear.
The logs cannot be found at the storage at it happens for windows. The file shares are empty.
I have Application Insights activated, but what I really need now is the application logs. I also tried activating Diagnostic Settings, selected Logs - > Function App Logs, and checked "Archive to a storage account".
But the storage account remains empty.

Is there a way to get them?

Thanks

**Comment by sjuarezgx** [comment]
I'm following this document with no results:
https://docs.microsoft.com/en-us/azure/azure-functions/functions-monitor-log-analytics?tabs=csharp

I haven't got the FunctionAppLogs table under LogManagement for my Log Analytics workspace.

**Comment by liliankasem** [comment]
Hello! I haven't been able to repro your issue. I deployed a linux dotnet v4 function app (not container) and following the documentation you linked I can see logs in Log Analytics and in the storage account:

**Storage:**

<img width="271" alt="storage" src="https://user-images.githubusercontent.com/2198905/144479477-dc90ee45-540b-4652-9cf2-75e121593a43.png">

**Log Analytics:**

<img width="993" alt="Screen Shot 2021-12-02 at 9 44 05 PM" src="https://user-images.githubusercontent.com/2198905/144508063-de18587e-c0cd-49bc-9d3d-6aa1d6b28780.png">

The function app I deployed: https://github.com/liliankasem/func-repro-7935

It does take 5-10 minutes for the logs to start populating. Are they still not showing up even now? It might be worth sharing a repository with the minimal code to repro your issue so we can investigate further.
